packages/forms/src/elements/tiles/Tiles.tsx

```diff
@@ -46,7 +47,7 @@ interface IStaticTilesExport<T, P>
  * @extends HTMLAttributes<HTMLDivElement>
  */
 export const Tiles = React.forwardRef<HTMLDivElement, ITilesProps>(
-  ({ onChange, value: controlledValue, isCentered, ...props }, ref) => {
+  ({ onChange, value: controlledValue, name, isCentered, ...props }, ref) => {
```

Comment:
This fixes an issue where `name` wasn't being applied to the tile input fields.

Reply:
This should have been a major version change...it was causing breaking builds for anyone who had a linter in place (or any SSR build process)